src/Form/FenceReviewCaseType.php

```diff
@@ -42,6 +42,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('bringerIdentification', IdentificationType::class, [
                 'label' => $this->translator->trans('Bringer', [], 'case'),
+                'is_required' => true,
```

Comment:
In the future it may be worth considering having a service decide if a field is required or not, e.g. based on some context (that we can also use for validation).